.github/workflows/release.yml

```diff
@@ -72,3 +72,7 @@ jobs:
         yarn publish:release 
         npx sleep 5
         yarn build:containers:release
+      env:
+          GH_TOKEN: ${{ secrets.GITHUB_PERSONAL_ACCESS_TOKEN }}
```

Comment:
Maybe `GITHUB_TOKEN` var will do the job?

https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Reply:
That's actually a good point. Possibly the issue is that GITHUB_TOKEN is not assigned to the action:

https://docs.github.com/en/actions/reference/authentication-in-a-workflow#example-passing-github_token-as-an-input

Reply:
I was thinking to set `${{ secrets.GITHUB_TOKEN }}` instead of `${{ secrets.GITHUB_PERSONAL_ACCESS_TOKEN }}` :)

```yaml
      env:
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          ```

Reply:
That shouldn't make a difference, `lerna publish` should pick up `secrets.GITHUB_TOKEN` if it was available. But I think it isn't - check my link above.